python/cudf/cudf/_lib/sort.pyx

```diff
@@ -129,17 +131,19 @@ cdef pair[vector[order], vector[null_order]] ordering(
     -------
     pair of vectors (order, and null_order)
     """
-    cdef vector[order] c_column_order
+    cdef vector[cpp_order] c_column_order
     cdef vector[null_order] c_null_precedence
     for asc, null in zip(column_order, null_precedence):
-        c_column_order.push_back(order.ASCENDING if asc else order.DESCENDING)
+        c_column_order.push_back(
```

Comment:
I dislike that we use c_ and cpp_ in different places here…

Reply:
I agree, it's not great, but I don't think it's worth addressing here. All of these bits of code will get rewritten in pylibcudf so we should aim to do a better job from the get-go with those versions instead.